test/Application.test.tsx

```diff
@@ -3,3 +3,7 @@ import React from 'react';
 import { expect } from 'chai';
 import { shallow } from 'enzyme';
 // import Application from './../src/app/components/Application/Application.jsx';
+
+describe("Application", () => {
+  it("should pass", () => {});
```

Comment:
Why is the test empty?

Reply:
The file originally had no tests. Mocha didn't complain but jest does if you have an empty test file. So added a temporary empty test.